### ISSUE TITLE
Warning by importing photo_view_scale_state.dart

### ISSUE THREAD

**Issue by kitoko552** [renancaraujo/photo_view#58]
`photo_view` is very nice library.
It helps me to show my app users photos easily.

I found an issue about importing `src/photo_view_scale_state.dart`.
I need to handle my app state by using PhotoViewScaleState and I'm using `scaleStateChangedCallback` to do that.
I need `import 'package:photo_view/src/photo_view_scale_state.dart';` to use PhotoViewScaleState in my project code, but if do so, I get warning from Android Studio like this.

<img width="545" alt="2018-11-26 11 38 37" src="https://user-images.githubusercontent.com/8244046/48990470-ecb9d400-f171-11e8-8236-4ca190ec29bb.png">

I can build my app though.
Could you fix this?

Log of `flutter doctor` is here.
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel dev, v0.11.9, on Mac OS X 10.13.6 17G65, locale ja-JP)
[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
[✓] Android Studio (version 3.2)
[✓] IntelliJ IDEA Community Edition (version 2018.1.2)
[!] VS Code (version 1.27.2)
[✓] Connected device (1 available)
```

**Comment by renancaraujo** [comment]
You have `PhotoViewScaleState` only by importing `photo_view/photo_view.dart`

**Comment by renancaraujo** [comment]
Thanks for pointing out. Looking into it.

**Comment by renancaraujo** [comment]
Just published the fix  [0.1.0](https://github.com/renancaraujo/photo_view/releases/tag/0.1.0)
